docs/data/sql_connectors.yml

```diff
@@ -156,15 +156,15 @@ kafka:
     versions:
         - version: universal
           maven: flink-connector-kafka
-          sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/$version/flink-sql-connector-kafka-$version.jar
+          sql_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/4.0.0-2.0/flink-sql-connector-kafka-4.0.0-2.0.jar
```

Comment:
I am wondering why need to hard code the version.
I assume prior to this change the version was coming through as 3.4.0-1.20 or was this never working? 
If we have to hard code - it would make sense to introduce a new variable , maybe constructed from $version